pxr/usd/plugin/usdAbc/testenv/testUsdAbcIndexedProperties/baseline/indexedTextureCoordinates.def.usda

```diff
@@ -53,8 +53,8 @@ def Mesh "Cube"
     uchar xform:ops.timeSamples = {
         0: 48,
     }
-    custom matrix4d xform:vals
-    matrix4d xform:vals.timeSamples = {
+    custom matrix4d xform
+    matrix4d xform.timeSamples = {
```

Comment:
I guess it's independent of this change, but this change in output hilites a problem we should probaly catch and fix:  if alembic allows arbitrary properties to be indexed, we can't support that, since USD only allows indexing of primvars.  So if we find anything other than an arbGeom property that's indexed, we should flatten it on the Alembic side?

Reply:
I think this points to two more problems:
 - in this case "vals" shouldn't be removed as there are no indices
 - the xform attribute isn't really used in this prim